Source/WebKit/WebProcess/WebPage/WebPage.h

```diff
@@ -2180,6 +2182,7 @@ class WebPage : public API::ObjectImpl<API::Object::Type::BundlePage>, public IP
     RefPtr<WebPageGroupProxy> m_pageGroup;
 
     String m_userAgent;
+    bool m_hasCustomUserAgent;
```

Comment:
Let's initialize this to `{ false }`

Reply:
Fixed in [ad1bffc](https://github.com/WebKit/WebKit/commit/ad1bffc9e4ac7ed1476417366186fa4010b047d7)